Windows CI failed

One of the windows CI tests is failing: https://github.com/ComputationalRadiationPhysics/alpaka/pull/948#issuecomment-603947472 

```
2020-03-25T15:53:50.4977214Z   CMake Warning at cmake/alpakaCommon.cmake:505 (message):
2020-03-25T15:53:50.5233265Z     MSVC 19.16.27038.0 does not support -G with CUDA! Device debug symbols NOT
2020-03-25T15:53:50.5234379Z     added.
2020-03-25T15:53:50.5235509Z   Call Stack (most recent call first):
2020-03-25T15:53:50.5235684Z     CMakeLists.txt:77 (include)
2020-03-25T15:53:50.5235799Z   
2020-03-25T15:53:50.5334428Z   
2020-03-25T15:53:50.5554698Z   -- ALPAKA_ACC_GPU_CUDA_ONLY_MODE
2020-03-25T15:53:50.6063841Z   -- ALPAKA_ACC_GPU_CUDA_ENABLED
2020-03-25T15:53:50.6758153Z   -- Catch2: Using INTERNAL version 2.11.0
2020-03-25T15:53:50.6775746Z   -- Catch2: Using INTERNAL version 2.11.0
2020-03-25T15:53:50.6904263Z   CMake Error at C:/Program Files/CMake/share/cmake-3.16/Modules/FindCUDA.cmake:1602 (configure_file):
2020-03-25T15:53:50.7069500Z     configure_file Problem configuring file
2020-03-25T15:53:50.7096867Z   Call Stack (most recent call first):
2020-03-25T15:53:50.7216230Z     C:/Program Files/CMake/share/cmake-3.16/Modules/FindCUDA.cmake:1868 (CUDA_WRAP_SRCS)
2020-03-25T15:53:50.7216454Z     cmake/addExecutable.cmake:37 (CUDA_ADD_EXECUTABLE)
2020-03-25T15:53:50.7551005Z     test/integ/cudaOnly/CMakeLists.txt:15 (alpaka_add_executable)
2020-03-25T15:53:50.7912375Z   
2020-03-25T15:53:50.8000778Z   
2020-03-25T15:53:50.9655282Z   -- Configuring done
2020-03-25T15:53:50.9656687Z   -- Configuring incomplete, errors occurred!
2020-03-25T15:53:50.9656881Z   See also "D:/a/alpaka/alpaka/build/CMakeFiles/CMakeOutput.log".
2020-03-25T15:53:50.9714568Z   See also "D:/a/alpaka/alpaka/build/CMakeFiles/CMakeError.log".
```

CC-ing: @BenjaminW3

--- a/script/run_build.sh
+++ b/script/run_build.sh
@@ -118,7 +118,7 @@ then
     make VERBOSE=1
 elif [ "$TRAVIS_OS_NAME" = "windows" ]
 then
-    "$MSBUILD_EXECUTABLE" "alpaka.sln" -p:Configuration=${CMAKE_BUILD_TYPE} -maxcpucount:2 -verbosity:minimal
+    "$MSBUILD_EXECUTABLE" "alpaka.sln" -p:Configuration=${CMAKE_BUILD_TYPE} -maxcpucount:1 -verbosity:minimal
 fi
 
 cd ..

